### ISSUE TITLE
merge react-web and react-native-web ? 

### ISSUE THREAD

**Issue by chandu0101** [taobaofed/react-web#87]
Hey,
         First of all thanks for the great effort you guys put into this project , i am currently using this in my project its working great :) , https://github.com/necolas/react-native-web (@necolas) this project also trying to achieve same thing, any chance to merge both projects ? (just an idea!)

**Comment by cameron** [comment]
I just started using react-native-web today (thanks, @necolas!), but am curious about your project, too, as it seems to have similar API coverage and community attention. 

Since the APIs should be identical and the work involved is mostly just keeping up with react-native, this seems like a great opportunity to merge and distribute the work. There'd be very little opportunity for disagreement over what is to be built, as facebook will have made the decisions for us, and probably not too much shared code between components, even, which would make scaling the team even easier.

Of course, herding cats may not be what anyone here signed up for, so thanks for the effort thus far, regardless :)

**Comment by yuanyan** [comment]
Current react-web is offers more then react-native-web, i have no idea how to merge two projects.  @necolas

**Comment by MoOx** [comment]
There is also https://github.com/KodersLab/react-native-for-web (poke @mattiamanzati) and I really thing we should discuss about that to have a single solid package. Fragmentation is not a good idea.
I think @necolas is a busy man (he does not do a lot of open source lately) but that would be nice to discuss about that, with @mattiamanzati, on how can we merge those 3 projects.

React ecosystem is a strength of React and we should merge effort to provide a good DX like the rest of the React ecosystem.

**Comment by grigio** [comment]
Yeah it's a pity, 3 projects with the same goal and none support React 15.x

**Comment by MoOx** [comment]
I am curious to know what people like @vjeux think about the state of react-native for web (and maybe the idea behind).

**Comment by MoOx** [comment]
@grigio react-native-web works with v15, there is some minors issues, but no big deal

**Comment by grigio** [comment]
@MoOx https://github.com/necolas/react-native-web/issues/123 it isn't a minor issue with React 15.x

**Comment by vjeux** [comment]
I haven't yet used react-native-web so I can't comment on the state of the project. On the direction, this is absoluely something that I want to exist. We haven't yet had a use case for it inside of Facebook so haven't really focused on it yet. But within the next year, I'm hoping that we can properly focus on it.

**Comment by johndangerstorey** [comment]
@vjeux I'm sure you're a busy guy and have a million higher priorities.... BUUUUUTTTT I would love it if this was built into react-native.  just sayin  😸

**Comment by vjeux** [comment]
@johndangerstorey this is absolutely something that I want to happen. In practice, it's not as easy as merging the project into react-native core unfortunately and will likely going to need someone full time to drive the effort in order to make it successful.

**Comment by johndangerstorey** [comment]
@vjeux right on.  I understand it's a "learn once" not "write once" philosophy... but still, it'd be nice to have that which CAN be written once to just be written once :)  But yeah, I'm sure it's a big project, not just like a "oh yeah, i'll just quickly merge that in" sort of thing.  Thanks for taking the time to respond.  Love the ideology and will be keeping my eye on this.  Exciting stuff.
